packages/datagateway-search/src/card/datasetSearchCardView.component.tsx

```diff
@@ -28,12 +21,19 @@ import {
   ISISDatasetDetailsPanel,
   DLSDatasetDetailsPanel,
   DatasetDetailsPanel,
+  useLuceneSearchInfinite,
+  SearchResponse,
+  SearchResultSource,
+  usePushDatasetFilter,
+  FiltersType,
+  formatBytes,
 } from 'datagateway-common';
 import { useTranslation } from 'react-i18next';
 import { useHistory, useLocation } from 'react-router-dom';
+import { styled, Paper, Typography } from '@mui/material';
 import { useSelector } from 'react-redux';
 import { StateType } from '../state/app.types';
-import { styled } from '@mui/material';
+import { CVCustomFilters } from 'datagateway-common/lib/card/cardView.component';
```

Comment:
I think it's better to export the `CVCustomFilters` type directly from datagateway-common as a whole instead of importing it like this? So maybe best to remove this line and add this  in `datagateway-common/src/index.tsx`:
```
export type { CVCustomFilters } from './card/cardView.component';
```